### PR TITLE
Fatima:Library Book

### PR DESCRIPTION
**Pull request by FatimaFarah93**
Completed exercise around the library book by creating a Book class and following the steps.

### CODE REVIEW

File: src/main/java/session6classes/LibraryBookFatima.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package session6classes;
+
+public class LibraryBookFatima {
+
+    public static class Book {
```
Comment (owynowyn) [inline src/main/java/session6classes/LibraryBookFatima.java:5]: I think you could remove this entire line and the closing `}` on line 26 and just have all the code within the `LibraryBookFatima` class. May need to rename the constructor too and the instantiations in the main method

Comment (owynowyn) [reply src/main/java/session6classes/LibraryBookFatima.java:5]: Summed up, I dont think the internal static class is essential and I think it should be fine to just have your `LibraryBookFatima` class with all the code within it

**Review by owynowyn — APPROVED** [review]

**Review by owynowyn — CHANGES_REQUESTED** [review]
Again, minor changes to make as I dont think the internal class is necessary. Let me know if you want to run through anything or need any assistance with it! Great work again though!

**Comment by FatimaFarah93** [comment]
Hi @owynowyn I've made the changes and moved the code into the internal class, should be in the latest version of the branch as I've pushed it all up now

**Review by owynowyn — APPROVED** [review]
Great work again Fatima

**Pull request merged**